include/deal.II/base/numbers.h

```diff
@@ -37,8 +37,55 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+namespace internal
+{
+  template <typename Number>
+  struct VectorizedArrayWidthSpecifier
```

Comment:
Can you add documentation to this class and to the specializations below?

That's because in some sense, this class may be in an internal namespace, but it's in the public interface as it provides the default template argument for the `VectorizedArray` class, and so should be documented.